chap43/chap43.tex

```diff
@@ -42,14 +43,14 @@ \subsection{奖励可以满足短期和长期的监管和非监管需求}
 
 进食、饮水和体温调节行为及其潜在的动机状态通常是响应（或预期）生理失衡而出现的。
 在这些情况下，行动会在相对较短的时间内获得奖励。
-相比之下，一些动机状态服务于除了短期生理稳态之外的生物学需要。
+相比之下，一些动机状态服务于除了短期生理稳态之外的生理需要。
 更复杂的长期目标，例如寻找和维持爱情伴侣或实现教育或职业目标，需要在更长的时间尺度上以目标为导向的行动。
 非调节性动机状态可能类似于生理信号产生的动机状态，但动机行为通常涉及一系列行动，其中并非每一个行动都会立即得到奖励（除了在朝着长期目标取得进展的意义上）。
 
 
-一般而言，激励刺激，甚至是仅表明朝着长期目标取得进展的刺激，都可以影响动机状态，从而完成复杂的行为序列。
-这个概念的一个简单例子是，猎豹必须跟踪、追逐、跑下并杀死羚羊，然后在开始进食之前将尸体拖到避难所。
-当然，即使是觅食和喂养所涉及的复杂动作，也远比有动机获得研究生学位和发展学术生涯的学生所需的步骤简单得多。
+总的来说，激励刺激，甚至是仅表明进展向着长期目标的刺激，都可以影响动机状态，从而完成复杂的行为序列。
```

Comment:
`进展向着长期目标的刺激`感觉读不通。
原文：“In general, incentive stimuli, even stimuli that only signal progress toward a longer-term goal, can influence motivational states so that complex behavioral sequences are completed.”

Reply:
已修改